example/workflows.yml

```diff
@@ -658,7 +658,7 @@ KinesisTriggerTest:
         - ErrorEquals:
             - States.ALL
           IntervalSeconds: 10
-          MaxAttempts: 3
+          MaxAttempts: 0
```

Comment:
Should we take the retry out then?

Reply:
I can.  I was just dropping it down to 0 so that we could see failures, but I'll take it out.

Reply:
https://github.com/nasa/cumulus/pull/565/commits/22f70b454eb3f771c1376f55011d2b5cdafba4d6